.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
```

Comment:
Maybe we should pin this to 3.8 since this is something we run very rarely and really wouldn't want to break?

Reply:
Fine by me ;-)